package.json

```diff
@@ -95,10 +93,12 @@
     "fastclick": "^1.0.6",
     "file-saver": "^2.0.5",
     "font-awesome": "^4.7.0",
+    "helmet-csp": "3.4.0",
```

Comment:
That was my fault. In my first attempt, I initially used yarn to make the dependency change and it was upgrading the packages on `add`. I used a specific version in this second attempt to avoid it and it removed the caret. I should have just moved it manually from devDependencies to dependencies in my first attempt. Just pushed a new commit @aeharding. Thanks for your patience.